src/gui/torrentcontentmodel.cpp

```diff
@@ -284,14 +284,15 @@ bool TorrentContentModel::setData(const QModelIndex &index, const QVariant &valu
     {
         auto *item = static_cast<TorrentContentModelItem*>(index.internalPointer());
         qDebug("setData(%s, %d)", qUtf8Printable(item->name()), value.toInt());
-        if (static_cast<int>(item->priority()) != value.toInt())
-        {
-            BitTorrent::DownloadPriority prio = BitTorrent::DownloadPriority::Normal;
-            if (value.toInt() == Qt::PartiallyChecked)
-                prio = BitTorrent::DownloadPriority::Mixed;
-            else if (value.toInt() == Qt::Unchecked)
-                prio = BitTorrent::DownloadPriority::Ignored;
 
+        BitTorrent::DownloadPriority prio = BitTorrent::DownloadPriority::Normal;
```

Comment:
Not intended for this PR, just some comment for a future redesign:
This line probably causes the transition from "unchecked" to "checked" items to always have Normal download priority and IMO it should preserve the previous download priority instead (High, Maximum, etc).